cub/cub/device/dispatch/tuning/tuning_reduce.cuh

```diff
@@ -366,6 +385,26 @@ struct policy_hub
     template <typename Tuning>
     static auto select_agent_policy(long) -> typename Policy600::ReducePolicy;
 
+    using ReducePolicy = decltype(select_agent_policy<sm86_tuning<AccumT>>(0));
+
+    // SingleTilePolicy
+    using SingleTilePolicy = ReducePolicy;
+  };
+
+  /// SM90
+  struct Policy900 : ChainedPolicy<900, Policy900, Policy860>
+  {
+    static constexpr int items_per_vec_load = 4;
+
+    // Use values from tuning if a specialization exists, otherwise pick Policy600
+    template <typename Tuning>
+    static auto select_agent_policy(int)
+      -> AgentReducePolicy<Tuning::threads, Tuning::items, AccumT, items_per_vec_load, BLOCK_REDUCE_RAKING, LOAD_LDG>;
+
+    // use Policy600 as DefaultPolicy
+    template <typename Tuning>
+    static auto select_agent_policy(long) -> typename Policy860::ReducePolicy;
```

Comment:
```suggestion
    // Use values from tuning if a specialization exists, otherwise pick Policy860
    template <typename Tuning>
    static auto select_agent_policy(int)
      -> AgentReducePolicy<Tuning::threads, Tuning::items, AccumT, items_per_vec_load, BLOCK_REDUCE_RAKING, LOAD_LDG>;

    // use Policy860 as DefaultPolicy
    template <typename Tuning>
    static auto select_agent_policy(long) -> typename Policy860::ReducePolicy;
```